src/sql/plot.py

```diff
@@ -328,9 +329,10 @@ def histogram(payload, table, column, bins, with_=None, conn=None):
     .. plot:: ../examples/plot_histogram_many.py
     """
     ax = plt.gca()
-    payload[
+    if sql.connection.Connection.current:
+        payload[
             "connection_info"
-        ] = sql.connection.Connection._get_curr_connection_info()
+        ] = sql.connection.Connection.current._get_curr_connection_info()
```

Comment:
related to the comments above. this became a classmethod in a recently merged PR


```suggestion
        ] = sql.connection.Connection._get_curr_connection_info()
```